CHANGELOG.rst

```diff
@@ -1,3 +1,10 @@
+**JDBC Driver 3.13.15**
+
+- \| | Refactored the isFileTransfer function into the base class.
```

Comment:
Should this also include the following: 
1. Updated version for Apache Tika
2. Allow URLs with underscores by default, add configuration to covert underscores to hyphens.